src/plugins/dashboard/public/application/application.ts

```diff
@@ -45,9 +45,10 @@ import {
   AppMountParameters,
 } from 'opensearch-dashboards/public';
 import { UsageCollectionSetup } from 'src/plugins/usage_collection/public';
+import { DashboardProvider } from 'src/plugins/dashboard/public/types';
 import { Storage } from '../../../opensearch_dashboards_utils/public';
 // @ts-ignore
-import { initDashboardApp } from './legacy_app';
+import { DashboardListItem, DashboardListProviderFn, initDashboardApp } from './legacy_app.js';
```

Comment:
Was this meant to be changed to `'./legacy_app.js'`? Shouldn't it continue working as `'./legacy_app'` or am I missing something?

Reply:
Interesting catch.  IDE did that at some point after some file refactoring.

Fixed.